src/static/riot/competitions/editor/_competition_details.tag

```diff
@@ -94,10 +157,16 @@
 
             self.refs.title.value = competition.title
             self.markdown_editor.value(competition.description || '')
+            self.refs.queue.value = _.get(competition, 'queue.id', null)
+            $('.queue-search').dropdown('set text', _.get(competition, 'queue.name', null))
+            $('.queue-search').dropdown('set value', _.get(competition, 'queue.id', null))
 
             // Value comes like c:/fakepath/file_name.txt -- cut out everything but file_name.txt
-            self.uploaded_logo_name = competition.logo.replace(/\\/g, '/').replace(/.*\//, '')
-            self.uploaded_logo = competition.logo
+            // TODO: Added this because my form was not receiving a logo from compeititon.
```

Comment:
Did the competition have a logo? is this going to mask a different error?

Reply:
The competition did have a logo.

I saw a comment I believe in regards to this:
```
            // Require title, logo is optional IF we are editing -- will just keep the old one if
            // a new one is not provided
```

But it seems like this is something to look into

Reply:
Can you make a github issue with information on how to replicate the error you were getting so that we can debug this?

Reply:
Can do

Reply:
✔️